.ci/install_runtime.sh

```diff
@@ -77,7 +78,14 @@ case "${KATA_HYPERVISOR}" in
 		;;
 esac
 
-
+# Replace the image created at the rootfs for the rust agent
+if [ "${TEST_RUST_AGENT}" == true ]; then
+	osbuilder_repo="github.com/kata-containers/osbuilder"
+	echo "Move rust image to /usr/share/kata-containers"
+	sudo mv "${GOPATH}/src/${osbuilder_repo}/kata-containers.img" "/usr/share/kata-containers/"
+	echo "Removing initrd image from configuration.toml"
+	sudo sed -i -e '/^initrd =/d' "${runtime_config_path}"
```

Comment:
iirc, by default, `initrd` is commented, so we shouldn't need to remove the line.

Reply:
@chavafg that is weird because when I ran the scripts both `image` and `initrd` were generated in the configuration.toml, that is the reason why I am removing the `initrd`

Reply:
you're right, but the script already handles this below, so we don't need to make a special case for `TEST_RUST_AGENT`

see:
https://github.com/kata-containers/tests/blob/20a64cf98f777362aa9e7e21e51af4a774335702/.ci/install_runtime.sh#L98-L104

Reply:
@chavafg so let me move that because that needs to be removed before doing https://github.com/kata-containers/tests/blob/20a64cf98f777362aa9e7e21e51af4a774335702/.ci/install_runtime.sh#L87
or the script will fail

Reply:
@chavafg done